src/ImageMessage.ts

```diff
@@ -0,0 +1,12 @@
+import { TextMessage } from "./types";
```

Comment:
remove doublequotes and semicolon. Please check PR files before sending it. Im not a linter